### PR TITLE
Coordination meeting 2021-03-18

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+### November Workshop Survey and Outcomes
+
+See slides attached to the agenda, which give highlights of the survey from the workshop.
+
+- Feedback: no need for a single virtual workshop to be packed with disparate topics, so indeed perhaps better to split into smaller events.
+
+### Proposal for DUNE Framework Requirements Review
+
+Paul: DUNE is trying to workout how to deal with its data, including massive "events" from supernovae (100GB). DUNE task force has defined the requirements (43 of them) in a document, which is aimed at DUNE people. If the HSF is giving feedback on this, how can it be expanded/annotated to non-DUNE people? Once that's done can we get together a group of experts who could review and bring input from the expertise of the wider community?
```
Comment (paullaycock) [inline organization/_posts/2021/2021-03-18-coordination.md:40]: A supernova "event" is over 100TBs, believe it or not, it's 100 seconds of continuous readout.

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+- Makes a lot of sense to have the HSF do this, crosscheck requirements and make sure that nothing is forgotten
+- Could we ask people from astro-particle to be involved as well, especially to look at the SN event issues
+
+Andrew: wanted to define a timescale on which feedback was given - early enough to be able to take requirements and statements back to FAs, viz. 4-6 weeks... (timescale is driven by US funding and a computing TDR that is due in June).
```
Comment (paullaycock) [inline organization/_posts/2021/2021-03-18-coordination.md:45]: Andrew mis-spoke, it's the CDR (not TDR) that is due in summer

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+### November Workshop Survey and Outcomes
+
+See slides attached to the agenda, which give highlights of the survey from the workshop.
+
+- Feedback: no need for a single virtual workshop to be packed with disparate topics, so indeed perhaps better to split into smaller events.
+
+### Proposal for DUNE Framework Requirements Review
+
+Paul: DUNE is trying to workout how to deal with its data, including massive "events" from supernovae (100TB). DUNE task force has defined the requirements (43 of them) in a document, which is aimed at DUNE people. If the HSF is giving feedback on this, how can it be expanded/annotated to non-DUNE people? Once that's done can we get together a group of experts who could review and bring input from the expertise of the wider community?
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-03-18-coordination.md:40]:
```suggestion
Paul: DUNE is trying to work out how to deal with its data, including massive "events" from supernovae (100TB!). DUNE task force has defined the requirements (43 of them) in a document, which is aimed at DUNE people. If the HSF is to give feedback on this, how can it be expanded/annotated to non-DUNE people? Once that's done can we get together a group of experts who could review and bring input from the expertise of the wider community?
```

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+### Detector Simulation
+
+- Next two meetings scheduled: <https://indico.cern.ch/category/10916/>
+    - 12th April: [Using GPUs in detector simulations](https://indico.cern.ch/event/1019940/). Have updates on AdePT, Celeritas and Opticks/Geant4 examples. See this as opportunity for these projects to report on computing model/progress/plans, _and_ to solicit initial feedback from simulation _users_ on progress on these topics. We will advertise through the usual lists, but wider advertisement very welcome, especially outside of LHC experiments!
+    - 10th May: [Simulating solid-state microphysics with Geant4 and G4CMP](https://indico.cern.ch/event/1016632/)
+
+### Reconstruction and Software Triggers
+
+- Kick-off meeting evaluating the outcome of the survey held on Wednesday: https://indico.cern.ch/event/1011873/
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-03-18-coordination.md:89]: I can't remember now if URLs should be in between <...> or not? This one (and others) isn't, but 4 lines above the <...> syntax is used. Just act as/if necessary :-).

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+Had an open discussion a few weeks ago that went well. Current schedule is open for handing the anticipated DNUE review.
+
+---
+
+## AOB
+
+### Next Meeting(s)
+
+- Next meeting will be 1 April
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-03-18-coordination.md:153]:
```suggestion
- Next meeting will be 1 April - no joke!
```

**Review by eduardo-rodrigues — APPROVED** [review]
LGTM. I added a couple of suggestions.

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+- **Agenda:** https://indico.cern.ch/event/1003975/
+- **Topic:** Compiler optimization reverse engineering using [Cutter](https://cutter.re/)
+- **Presenter:** Hadrien Benjamin Grasland
+
+**Graeme:** there was a very interesting discussion about Spack in the SFT LIM meeting on Monday. It would be really useful to hear from the Fermilab Spack people in the WG soon.
+- **Agenda:** <https://indico.cern.ch/event/1016908/>
+
+### Software Training
+
+Please contact us or drop into weekly meeting Monday 16h CET so we can help you organise a training.
```
Comment (amete) [inline organization/_posts/2021/2021-03-18-coordination.md:133]:
```suggestion
Please contact us or drop into the weekly meeting Monday 16h CET so we can help you organise a training.
```

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+### HL-LHC Review Phase 2
+
+We have had some meetings with key people for each section of the document. Now making progress in contacting other projects and arranging mini-workshops, as needed.
+
+### Google Summer of Code, Season of Docs
+
+The GSoC mentors are quite busy at the moment with candidate evaluation and there are few exchanges on the mentors list.
+
+For GSoD so far we have **no applications**. *Do no projects need documentation help?* If no proposals we won't apply. Deadline is 25 March.
```
Comment (amete) [inline organization/_posts/2021/2021-03-18-coordination.md:26]:
```suggestion
For GSoD so far we have **no applications**. *Do no projects need documentation help?* If there are no proposals we won't apply. Deadline is 25 March.
```

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+- Will send another reminder
+- Can we consider also facility areas that need documentation?
```
Comment (amete) [inline organization/_posts/2021/2021-03-18-coordination.md:29]:
```suggestion
- Can we also consider facility areas that need documentation?
```

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+Andrew: wanted to define a timescale on which feedback was given - early enough to be able to take requirements and statements back to FAs, viz. 4-6 weeks... (timescale is driven by US funding and a computing CDR that is due in June).
+
+- To crosscheck the requirements or to check how much existing software matches those (fixed) requirements? Would not be able to do both on a short timescale.
+
+Andrew: check on requirements should be a quick part; real question is how do those requirements fit into the HEP ecosystem?
+
+- We would try to leverage the broadest expertise possible, of people willing to help
+- Are the facility boundary conditions clear? GPUs or not? Dedicated or shared resources? Special facilities like analysis facilities?
+
+Andrew: there is a description of some of this and we do anticipate being quite free to decide how to use these resources; data sizes and form of a neutrino analysis are well known. Some feedback on this aspect would also be useful, when people come with different experience.
```
Comment (amete) [inline organization/_posts/2021/2021-03-18-coordination.md:54]:
```suggestion
Andrew: there is a description of some of this and we do anticipate being quite free to decide how to use these resources; data sizes and form of a neutrino analysis are well known. Some feedback on this aspect would also be useful, when people come with different experiences.
```

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+    - Yes, this is a good idea
+    - Chris: agree, but we do want the HSF to assemble a panel of suitable experts
+
+Benedikt, Chris, Kyle and Attila will discuss then how to proceed.
+
+## Working Group Updates
+
+### Data Analysis
+
+- Work has started on metadata document. After the skeleton document has been created will ask metadata meeting contributors to help populate it
```
Comment (amete) [inline organization/_posts/2021/2021-03-18-coordination.md:70]:
```suggestion
- Work has started on the metadata document. After the skeleton document has been created will ask metadata meeting contributors to help populate it
```

File: organization/_posts/2021/2021-03-18-coordination.md [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+    - BNL (RHIC/EIC)
+    - JLab
+    - FAIR
+    - LZ
+    - ISOLDE?
+
+### Detector Simulation
+
+- Next two meetings scheduled: <https://indico.cern.ch/category/10916/>
+    - 12th April: [Using GPUs in detector simulations](https://indico.cern.ch/event/1019940/). Have updates on AdePT, Celeritas and Opticks/Geant4 examples. See this as opportunity for these projects to report on computing model/progress/plans, _and_ to solicit initial feedback from simulation _users_ on progress on these topics. We will advertise through the usual lists, but wider advertisement very welcome, especially outside of LHC experiments!
```
Comment (amete) [inline organization/_posts/2021/2021-03-18-coordination.md:84]:
```suggestion
    - 12th April: [Using GPUs in detector simulations](https://indico.cern.ch/event/1019940/). Have updates on AdePT, Celeritas and Opticks/Geant4 examples. See this as an opportunity for these projects to report on computing models/progress/plans, _and_ to solicit initial feedback from simulation _users_ on progress on these topics. We will advertise through the usual lists, but wider advertisement is very welcome, especially outside of LHC experiments!
```

**Review by amete — APPROVED** [review]
I have only a few minor comments but otherwise it LGTM. Thanks a lot @graeme-a-stewart.

File: organization/_posts/2021/2021-03-18-coordination.md (hunk 4 above)
Comment (graeme-a-stewart) [reply organization/_posts/2021/2021-03-18-coordination.md:89]: Oh, yes, one does need the <>s - didn't spot this one, will fix.

File: organization/_posts/2021/2021-03-18-coordination.md (hunk 9 above)
Comment (graeme-a-stewart) [reply organization/_posts/2021/2021-03-18-coordination.md:54]: Here I see "experience" as used in an abstract, uncountable, way - for me it's more natural like this.

File: organization/_posts/2021/2021-03-18-coordination.md (hunk 11 above)
Comment (graeme-a-stewart) [reply organization/_posts/2021/2021-03-18-coordination.md:84]: It is just the "computing model" of each individual project

**Comment by graeme-a-stewart** [comment]
Thanks all - now good to merge!

**Pull request merged**